<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaInfoPersonas</class>
 <widget class="QDialog" name="VentanaInfoPersonas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tablaPersonas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>1001</width>
     <height>451</height>
    </rect>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>35</number>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Apellido</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>País</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Creencia</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Profesión</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Correo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nacimiento</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Hijos/Hijas</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Lujuria</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Gula</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Avaricia</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Pereza</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ira</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Envidia</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Soberbia</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Suma Pecados</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="filtProfesiones">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filtPaises">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filtCreencias">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filtApellidos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="filtContinentes">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>30</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Paises</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Creencia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Continentes</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Profesiones</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>70</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>570</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mostrar todo</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
